packages/storybook/stories/Adapters/Adapters.stories.mdx

```diff
@@ -25,23 +25,22 @@ These methods are asynchronous so we will use `useEffect` to create them.
 
 ### Call Adapter
 
-To initialize the call adapter, you will be required to provide a call locator with the information of the group or meeting you are trying to join.
-Group Id for Azure Communication Services take the shape of GUID generated by the developer to identify a call inside their tenant.
+To initialize the call adapter, you will be required to provide a call locator with the information of the group call or meeting you are trying to join.
 Teams Meeting information can be acquired through [Microsoft Graph](https://docs.microsoft.com/graph/overview).
 
 <Source code={CallAdapterExampleText} />
 
 ### Chat Adapter
 
-To initialize the chat adapter, you will be required to provide an endpoint url for the Azure Communication Services resource on [Azure Portal](https://portal.azure.com).
+To initialize the chat adapter, you will be required to provide an endpoint from your Azure Communication Services resource on [Azure Portal](https://portal.azure.com).
```

Comment:
just curious: why removing 'url'?

Reply:
across our code we renamed endpointUrl to endpoint. I felt it might be good to rename it here and to not push that inplementation detail forward.